### ISSUE TITLE
Simple move_and_slide sprite code not working in 3.1 Beta 3/4

### ISSUE THREAD

**Issue by Bat-Computer** [godotengine/godot#25970]
**Godot Version**
Godot Beta 3.1 Beta 3 and Beta 4

**OS/device including version**
Windows 10 74bit, Geforce 550ti

**Issue description**
Code that is manually typed into the Beta 3 and 4 no long works. The Sprite no longer will move. However, if you import the Project from Stable 3.06 into Beta 3/4 all woks!

**Steps to reproduce**
I have included two projects, one written in Stable 3.06 and the other re-written in Beta 4 (affects Beta 3 also). They demonstrate a simple move_and_slide sprite. The 3.06 project works, but the re-written script in Beta 3/4 does not. If you import the original 3.06 project into the new Betas, in the conversion it all appears to work. I can't see any reference to what has changed if anything in the newer versions.

**Minimal reproduction project**
2 Project files included in a zip archive.
[Strange Bug help.zip](https://github.com/godotengine/godot/files/2872940/Strange.Bug.help.zip)

Basic GDScript Code:

```
extends KinematicBody2D

var player_velocity = Vector2()


func _physics_process(delta):
	
	if Input.is_action_pressed("ui_right"):
		player_velocity.x = 30
		
	move_and_slide(player_velocity)
```

**Dropbox Download**
https://db.tt/xhuwCIbTL0

**Comment by dramdrum** [comment]
It works if you add a CollisionShape2D to the player.

**Comment by akien-mga** [comment]
Duplicate of #24775, fixed by f439b786ea4b317c7e30a978e1b61a9578f1f483 (should be included in the upcoming 3.1 beta 5).
